src/Symfony/Bundle/FrameworkBundle/Command/YamlLintCommand.php

```diff
@@ -24,6 +24,7 @@
 class YamlLintCommand extends BaseLintCommand
 {
     protected static $defaultName = 'lint:yaml';
+    protected static $defaultDescription = 'Lints a file and outputs encountered errors';
```

Comment:
descriptions borrowed from the parent

Reply:
should the `setDescription()` call in the parent be changed to use `static::` instead of `self`?

Reply:
Nope: we don't want to inherit here.

Reply:
Isn't it the opposite with the logic inside `getDefaultDescription`?
- getDefaultDescription return null if `$defaultDescription` is not overridden by the child
- `YamlLintCommand::configure`, let `BaseLintCommand` configuring the description which use its own description

leading to an inconsistency `getDefaultDescription` returns `YamlLintCommand`'s description
`$command->getDescrition()` returns the `LintCommand`'s description

Reply:
From a descriptive pov, this is expected yes.

The inconsistency is resolved by LazyCommand, which calls setDescription when initializing.